pydantic/_internal/_generate_schema.py

```diff
@@ -1542,19 +1534,16 @@ def _apply_annotations(  # noqa: C901
         pydantic_js_annotation_functions: list[GetJsonSchemaFunction] = []
 
         def inner_handler(obj: Any) -> CoreSchema:
-            if isinstance(obj, type(Annotated[int, 123])):
-                schema = transform_inner_schema(self._annotated_schema(obj))
```

Comment:
this branch no longer seemed necessary now that `_generate_schema` handles `Annotated` properly (and `Annotated` doesn't hit any of the other property extractions as far as I can tell).